MBBSEmu.Tests/Integration/MBBSEmuIntegrationTestBase.cs

```diff
@@ -66,11 +66,9 @@ protected void ExecuteTest(TestLogic testLogic)
 
             CopyModuleToTempPath(ResourceManager.GetTestResourceManager());
 
-            var modules = new List<MbbsModule>
-            {
-                new MbbsModule(serviceResolver.GetService<IFileUtility>(), serviceResolver.GetService<ILogger>(), "MBBSEMU", _modulePath)
-            };
-
+            new MbbsModule(serviceResolver.GetService<IFileUtility>(), serviceResolver.GetService<ILogger>(), "MBBSEMU",
+                _modulePath);
```

Comment:
still not needed though?

Reply:
Here is what Host does:
```
 //Load Modules
            foreach (var m in moduleConfigurations)
                AddModule(new MbbsModule(_fileUtility, _logger, m.ModuleIdentifier, m.ModulePath) {MenuOptionKey = m.MenuOptionKey});
```

Basically MBBSModule initialize the module (loads msgs/mcvs/etc.) and AddModule activates it so to speak.... so yes we do need to call MBBSModule, and then in Host we call "AddModule" to initialize it